sphinx/writers/latex.py

```diff
@@ -1378,19 +1360,25 @@ def depart_hlistcol(self, node):
         pass
 
     def latex_image_length(self, width_str):
-        match = re.match('(\d*\.?\d*)\s*(\S*)', width_str)
+        return self.latex_dim_to_latexdim(width_str)
+
+    def latex_dim_to_latexdim(self, length_str):
```

Comment:
1. ``rstdim_to_latexdim`` looks good to me.  Should there be a prefix ?

2. because of ``\sphinxpxdimen`` the function makes sense only if used to write output into a LaTeX file using ``sphinx.sty``. Can we still define it as function at top level of `latex.py` ?